vittles/sensitivity_lib.py

```diff
@@ -284,9 +284,13 @@ def __init__(
         validate_optimum=False,
         hessian_at_opt=None,
         cross_hess_at_opt=None,
+        sens_mat=None,
         factorize_hessian=True,
         hyper_par_objective_fun=None,
-        grad_tol=1e-8):
+        grad_tol=1e-8,
+        system_solver=None,
```

Comment:
I think I'm going to re-do this PR, but in the future don't forget to update the documentation and tests!